doc/CCLUTMOCKING.md

```diff
@@ -187,34 +187,49 @@ call cclutRemoveAllMockImplementations(null)
 ```
 
 ## Implementation Notes
-1. For consistency, all mocking functions normalize names (tables, subroutines, records, etc.) to be uppercase.  This matches with CCL and Oracle.
+1. The mocking functions automatically normalize entity names to uppercase to match with CCL and Oracle.
 
-2. cclutRemoveAllMocks should be called as part of the teardown for all tests.  The framework will attempt to clean up any outstanding mocks, but it is good practice to explicitly remove any mocks to ensure that no mocked tables remain in the Oracle instance.
+1. `cclutRemoveAllMocks` should be called from teardown.  
```

Comment:
markdown ignores the provided number and ignores them sequentially. This makes it easier to reorder them.

Reply:
So, I thought the GitHub markdown viewer didn't have that feature implemented because they were still showing as "1" for me, but I can see now that the actual issue seems to be that the numbering resets when using bullets:

![image](https://user-images.githubusercontent.com/44979171/54446423-e77a9800-4714-11e9-9d3c-841ca576693d.png)

Reply:
Oh snap. Thanks for noticing.